package/src/components/Attachment/Gallery.tsx

```diff
@@ -59,7 +59,7 @@ const styles = StyleSheet.create({
     flexWrap: 'wrap',
     overflow: 'hidden',
   },
-  imageContainer: { padding: 1 },
+  imageContainer: { display: 'flex', flexDirection: 'row', justifyContent: 'center', padding: 1 },
```

Comment:
Huh, I'm surprised our linter doesn't make this break down into multiple lines 🤔

Reply:
I don't think this is a linting issue. If I add one more property it goes to multiline.